components/tools/OmeroWeb/test/integration/test_api_projects.py

```diff
@@ -395,17 +395,16 @@ def test_marshal_projects_pagination(self, user1, user2,
         version = settings.API_VERSIONS[-1]
         request_url = reverse('api_projects', kwargs={'api_version': version})
 
-        # First page, just 2 projects. Page = 1 by default
+        # First page, just 2 projects. offset = 0 by default
         limit = 2
         rsp = _get_response_json(django_client, request_url, {'limit': limit})
         assert len(rsp['data']) == limit
         assert_objects(conn, rsp['data'], projects[0:limit])
 
         # Check that page 2 gives next 2 projects
-        page = 2
-        payload = {'limit': limit, 'page': page}
+        payload = {'limit': limit, 'offset': limit}
```

Comment:
just to avoid possible mistake or miss some errors, I will use different values e.g. limit 2 projects 3 instead of 2/2